quill-spark/src/main/scala/io/getquill/QuillSparkContext.scala

```diff
@@ -84,7 +84,7 @@ trait QuillSparkContext
       node.structField.dataType match {
         case st: StructType =>
           // Recursively convert all parent array columns to single null values if all their children are null
-          val preculatedColumn = struct(node.children.map(percolateNullArraysRecursive(_)): _*)
+          val preculatedColumn = struct(node.children.map(percolateNullArraysRecursive(_)).toIndexedSeq: _*)
```

Comment:
Just curious, why is the `.toIndexedSeq` needed here?

Reply:
Deprecation warning under Scala 2.13:
`Passing an explicit array value to a Scala varargs method is deprecated (since 2.13.0) and will result in a defensive copy; Use the more efficient non-copying ArraySeq.unsafeWrapArray or an explicit toIndexedSeq call`